### DISCUSSION TITLE
⛔ 🛟 Support Request: WooCommerce Cart Page Loading Issue

### DISCUSSION THREAD

**Discussion by go-b-wedesign** [elementor/elementor#24830] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi Elementor Support.,

Great day. Hope you’re doing well.

I’m using the latest version of Elementor (3.18.3) and facing an issue with that. While using the function $elementor_instance->frontend->get_builder_content_for_display( $id ) it seems the error on the function it doesn’t allow me to view the cart page and products are not showing up. The products are successfully added to the cart but the woocommerce cart page is empty.

We investigated the woocommerce plugin side as well and installed fresh WordPress and woocommerce as well. But the issue still persisted. After deactivating the elementor the cart page is loading and working fine. So, we analyzed the elementor and we found the function (get_builder_content_for_display) had an issue. Could you please check this issue and let me know it will be more helpful for us.

If I downgraded the Elementor version to 3.17.3 the page is loading fine. Hope you’re clear with my explanation. If the explanation is not clear please let me know will add some screenshot (or) short video.

Thanks in Advance 🙂

### Describe the solution you'd like.

Hi Elementor Support.,

Great day. Hope you’re doing well.

I’m using the latest version of Elementor (3.18.3) and facing an issue with that. While using the function $elementor_instance->frontend->get_builder_content_for_display( $id ) it seems the error on the function it doesn’t allow me to view the cart page and products are not showing up. The products are successfully added to the cart but the woocommerce cart page is empty.

We investigated the woocommerce plugin side as well and installed fresh WordPress and woocommerce as well. But the issue still persisted. After deactivating the elementor the cart page is loading and working fine. So, we analyzed the elementor and we found the function (get_builder_content_for_display) had an issue. Could you please check this issue and let me know it will be more helpful for us.

If I downgraded the Elementor version to 3.17.3 the page is loading fine. Hope you’re clear with my explanation. If the explanation is not clear please let me know will add some screenshot (or) short video.

Thanks in Advance 🙂

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Prithika-Wedesign** [comment]
Hi There.,

I'm also facing same issue on the WooCommerce cart page. Is that any solution for this issue??

Let me know if any one got the solution, please...

**Comment by nicholaszein** [comment]
Hell,

Thanks for contacting us! 🙏 

### 🛟 This is more of a support question, not really something we can help with here. 

👉 Please remember to check the [Help Center](https://elementor.com/help) or contact [Elementor Support](https://elementor.com/contact) before submitting an issue.


### ⚠️ Note that GitHub Discussions are reserved for Elementor 🚀 Feature Requests only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

### ⚠️ As a gentle reminder, **🛟 Support Request Discussions** may need to be closed without troubleshooting. 
Remember to always follow our **[🚀 Feature Request Guidelines](https://elemn.to/gh-new-feature-request)** to save time and ensure smooth troubleshooting.

Thank you for understanding! 🙏 

Kind regards
